packages/create-cloudflare/src/helpers/codemod.ts

```diff
@@ -55,7 +56,7 @@ export const parseFile = (filePath: string) => {
 		const fileContents = readFile(path.resolve(filePath));
 
 		if (fileContents) {
-			return recast.parse(fileContents, { parser }) as Program;
+			return recast.parse(fileContents, { parser }).program as Program;
```

Comment:
Nope, not really a bug fix just a dx improvement, that we returned was technically incorrect.

The only consumer of `parseFile` was  previously `transformFile`, and because of the way it interacts with `recast`, passing it an ast with the entire program vs just the program body are functionally equivalent. Since we didn't ever consume the result of `parseFile` directly, the annoying dx of it's former implementation or the type error was never apparent.

The introduction of `loadSnippets` made the issue apparent since it works more directly with the AST.

As for test coverage: E2E already gives us a pretty good signal here since a codemod failure would likely result in a un-deployable project. That being said, there's probably a case for more direct unit test coverage now that we're using it more broadly.